marketplace_docs/templates/Fabric/files/etc/update-motd.d/99-one-click

```diff
@@ -2,14 +2,12 @@
 #
 # Configured as part of the DigitalOcean 1-Click Image build process
 
-myip=$(hostname -I | awk '{print$1}')
 cat <<EOF
 ********************************************************************************
 
-This is the text that will be shown in the MOTD users see when they log in via
-SSH.  This copy should include links to documentation or product informaiton
-along with any information that would helpful to users.
+If you have any queries or issues see https://zulip.readthedocs.io/en/latest/production/install.html#troubleshooting.
+For enterprise plans and support visit https://zulipchat.com/plans/
```

Comment:
Not sure what the text here should be, but maybe the following?

Troubleshooting guide: https://zulip.readthedocs.io/en/stable/production/troubleshooting.html
Secure, maintain, and upgrade: https://zulip.readthedocs.io/en/stable/production/maintain-secure-upgrade.html
Enterprise plans and support: https://zulipchat.com/plans/

Reply:
This looks great except that it's probably going to generate too-long lines since this is a text format.  Maybe just

You may find the following resources helpful:
* https://zulip.readthedocs.io/en/stable/production/troubleshooting.html
* https://zulip.readthedocs.io/en/stable/production/maintain-secure-upgrade.html
* https://zulipchat.com/plans/

Reply:
How about this?
```
For troubleshooting, maintainance, and enterprise support plans, see:
https://zulip.readthedocs.io/en/stable/production/troubleshooting.html
https://zulip.readthedocs.io/en/stable/production/maintain-secure-upgrade.html
https://zulipchat.com/plans/
```